xclim/data/fr.json

```diff
@@ -168,6 +168,13 @@
     "title": "Jours mouillés",
     "abstract": "Nombre de jours où les précipitations sont au-dessus d'un seuil."
   },
+  "WETDAY_PROP": {
```

Comment:
@aulemahal here's a placeholder for the french metadata so you can add as a commit suggestion.

Reply:
Done